apps/condo/domains/help/schema/Help.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Generated by `createschema help.Help 'organization:Relationship:Organization:CASCADE; name?:Text; properties:Relationship:Property:SET_NULL; content:Text;'`
+ */
+
+const { Text, Relationship } = require('@keystonejs/fields')
+const { Wysiwyg } = require('@keystonejs/fields-wysiwyg-tinymce')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { dvAndSender } = require('@condo/domains/common/schema/plugins/dvAndSender')
+const access = require('@condo/domains/help/access/Help')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+
+const Help = new GQLListSchema('Help', {
+    schemaDoc: 'Help information related to a property or group of properties',
+    fields: {
+
+        organization: ORGANIZATION_OWNED_FIELD,
+
+        name: {
+            schemaDoc: 'Help name, for example LCD name',
+            type: Text,
+        },
+
+        properties: {
+            schemaDoc: 'Properties to which this help applies',
+            type: Relationship,
+            ref: 'Property',
+            many: true,
+        },
+
+        content: {
+            schemaDoc: 'Help information related to a property or group of properties',
```

Comment:
This documentation comment is already used for schema. Maybe to write like this:
"Textual content of help in HTML format".